config/example-extras.cfg

```diff
@@ -1965,7 +1965,9 @@
 #analog_range_kill_pin:
 #   The resistance range for a 'kill' button. Range minimum and maximum
 #   comma-separated values must be provided when using analog button.
-
+#x_offset:
+#   Set the horizontal offset value on SSD1306/SH1106 displays.
+#
```

Comment:
This removes the blank line between sections.  Please add the `x_offset` parameter after the `vcomh` parameter.  The description should also state that the default is 0.